Build fails for v2.17.0

```
FAILURE: Build failed with an exception.
 
* What went wrong:
Execution failed for task ':app:mergeFdroidReleaseNativeLibs'.
> Could not resolve all files for configuration ':app:fdroidReleaseRuntimeClasspath'.
   > Failed to transform arm64_v8a_release-1.0.0-1527ae0ec577a4ef50e65f6fefcfc1326707d9bf.jar (io.flutter:arm64_v8a_release:1.0.0-1527ae0ec577a4ef50e65f6fefcfc1326707d9bf) to match attributes {artifactType=android-jni, org.gradle.category=library, org.gradle.libraryelements=jar, org.gradle.status=release, org.gradle.usage=java-runtime}.
      > Execution failed for JetifyTransform: /home/demizo/.gradle/caches/modules-2/files-2.1/io.flutter/arm64_v8a_release/1.0.0-1527ae0ec577a4ef50e65f6fefcfc1326707d9bf/86458862dc48126fd27484be94105ff1cc4cc58c/arm64_v8a_release-1.0.0-1527ae0ec577a4ef50e65f6fefcfc1326707d9bf.jar.
         > Java heap space
```

If I read that last line correctly (and yes, it stops exactly there), your gradle settings might need some adjustments. If you can let me know the proper adjustments (to match yours – you probably adjusted that locally already and just forgot to commit and push that change), we can give it another try. For now, the green shield indicating a successfully confirmed [Reproducible Build](https://izzyondroid.org/about/security/ReproducibleBuilds/), cannot come up at IzzyOnDroid for this release.

Thanks in advance for your help!

--- a/android/gradle.properties
+++ b/android/gradle.properties
@@ -1,3 +1,3 @@
-org.gradle.jvmargs=-Xmx1536M
+org.gradle.jvmargs=-Xmx2g
 android.useAndroidX=true
 android.enableJetifier=false

